### ISSUE TITLE
GRPC-SSL Endpoint Documentation Outdated

### ISSUE THREAD

**Issue by alex-seto** [SeldonIO/seldon-core#3237]
Hello, I have been trying to configure certificates for an SSL endpoint using GRPC transport using the documentation at https://docs.seldon.io/projects/seldon-core/en/latest/examples/seldon_client.html. However, it appears the tutorial linked (https://istio.io/latest/docs/tasks/traffic-management/ingress/secure-ingress/) has been updated to a newer version and produces certificates in a different format/nomenclature than in the Seldon Docs. I have tried to use both tutorials to generate the certificates, both the new and the old one (https://istio.io/v1.3/docs/tasks/traffic-management/ingress/secure-ingress-sds/) but I haven't had any success. I was wondering if you could provide an updated method to authenticate an SSL endpoint connection for GRPC (and REST, but I have been able to work around by skipping this authentication providing empty credentials). 

Thank you!

**Comment by cliveseldon** [comment]
Can you clarify the exact issue you are seeing so we can understand the issue better. Maybe with an example?

**Comment by alex-seto** [comment]
Hello @cliveseldon ,

thank you for the response. There were two pieces to this issue. This first is more straight forward; the documentation at this link: https://docs.seldon.io/projects/seldon-core/en/latest/examples/seldon_client.html attempts to provide a solution for dealing with GRPC handoffs with SSL authentication. However, the link it provides for making certificates (blue text "Istio Secure Gateway SDS example") ![image](https://user-images.githubusercontent.com/46577558/121914821-9f39ee00-cd00-11eb-8fb9-6dc448690ea3.png)
 does not appear to be the correct tutorial for producing certificates that are used in the seldon docs.

The link leads to the newest version of Istio docs which produce certificates which do not line up (in terms of file structure, nomenclature, etc) with ones listed in seldon docs. Instead, it appears the intended tutorial can be found at Istio v1.3 docs (outdated) as it provides a script for producing certificates in a similar manner to the ones which appear in the seldon tutorial.

Nevertheless, I am wondering if there is a better/cleaner work around than using the example certificates when in need of SSL auth for GRPC hand offs as producing these certs in the example and then having a gateway called "httpbin.example.com" seems like a forced fix. I have attempted to copy both tutorials and followed instructions but I have had no success with authenticating the endpoint (I have been able to tap in using REST payload and bypassing SSL auth).

Thank you again for the response
